.circleci/scripts/linux_test_sanity.sh

```diff
@@ -4,8 +4,8 @@ source scripts/install_utils
 
 MODELS=("fastrcnn" "fcn_resnet_101" "my_text_classifier_v2" "resnet-18" "my_text_classifier_scripted_v2" "alexnet_scripted" "fcn_resnet_101_scripted"
```

Comment:
Should we include here the `deeplabv3_resnet_101` and `deeplabv3_resnet_101_scripted` models so that the sanity checks also run over the DeepLabV3 model?

Reply:
Postponed for the next iteration.